_sass/globals/grid.scss

```diff
@@ -7,6 +7,11 @@
   overflow-x: hidden;
   overflow-y: auto;
 
+
+  &.overflow-y-hidden {
```

Comment:
I'm not a fan of this approach, I think if this style is specific to the popover at mobile sizes, it should go under that selector.